postgres/src/main/scala/zio/sql/postgresql/PostgresModule.scala

```diff
@@ -227,6 +227,12 @@ trait PostgresModule extends Jdbc { self =>
       FunctionDef[Timestampz, Timestampz](FunctionName("make_timestamptz"))
     val Encode                      = FunctionDef[(Chunk[Byte], String), String](FunctionName("encode"))
     val Decode                      = FunctionDef[(String, String), Chunk[Byte]](FunctionName("decode"))
+    val Format0                      = FunctionDef[String, String](FunctionName("format")) // TODO: varargs
```

Comment:
varargs not available yet (see also 'concat' function)
Is there an open issue for varargs support? otherwise I would open an issue to track this (and might give it a try)

Reply:
Created issue here https://github.com/zio/zio-sql/issues/439